src/table/toolbar/table-toolbar.component.ts

```diff
@@ -31,7 +32,11 @@ export class TableToolbar {
 	@Input() set ariaLabel (value) {
 		this.actionBarLabel.next(value.ACTION_BAR);
 	}
+	@Input() set cancel (value) {
```

Comment:
This should be `cancelText` ... I think we're generally standardizing on `xText` for translations that appear in the UI and `xLabel` for accessible translations/text.